src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java

```diff
@@ -85,4 +86,34 @@ public void testUIElements() {
         Assert.assertTrue(uiElements.isOthersRadioButtonSelected(), "Others radio button was not selected!");
     }
 
+
+
+    @Test(description = "01")
+    @MethodOwner(owner = "ashchavinska")
+    public void verifyLoginPage(){
+        String username = "Rachel Green";
+        String password = "1234567890";
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        Assert.assertTrue(loginPage.isPageOpened(), "Login page isn't opened");
+        //verify fields are presented->they must be (name,password,male/female... already checked previously
+        Assert.assertFalse((loginPage.isMaleSexChecked() && loginPage.isFemaleSexChecked()), "Sex already checked");
```

Comment:
better use one assert for one method, its will be more informative when test will be failed

Reply:
done